nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.js

```diff
@@ -99,6 +103,14 @@ NfRegistry.prototype = {
         } else {
             self.router.navigateByUrl('login');
         }
+    },
+    showAboutDialog: function () {
+        this.nfRegistryApi.showRegistryAboutDialog();
+        this.matDialog.open(NfRegistryExplorerAbout, {
+            width: '550px',
+            height: '440px',
+            data: this.nfRegistryService.registry.version
+        });
     }
```

Comment:
```suggestion
    showAboutDialog: function () {
        this.nfRegistryApi.showRegistryAboutDialog();
    }
```

Now the `showAboutDialog` method is not being called anywhere. Suggestion to call this method in the template, which would then only need to call `this.nfRegistryApi.showRegistryAboutDialog();`.